### PR TITLE
Added register.fxml and RegisterController.java (ESFI-13)

### PR DESCRIPTION
**Pull request by ancae17**

### CODE REVIEW

File: src/main/java/employment/system/controllers/RegisterController.java [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+
+
+
+        } catch (IOException e) {
+            e.printStackTrace();
+        }
+
+    }
+
+    private boolean isEmailValid(String email) {
```
Comment (Chupa-panka) [inline src/main/java/employment/system/controllers/RegisterController.java]: Please delete this function.

File: src/main/java/employment/system/controllers/RegisterController.java [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+        return false;
+    }
+
+    private boolean isFieldEmpty(String firstName) {
```
Comment (Chupa-panka) [inline src/main/java/employment/system/controllers/RegisterController.java]: Please delete this function.

File: src/main/resources/register.fxml [hunk 3]
```diff
@@ -0,0 +1,55 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<?import javafx.geometry.*?>
+<?import javafx.scene.control.*?>
+<?import javafx.scene.layout.*?>
+<?import javafx.scene.text.*?>
+
+<GridPane alignment="center" hgap="10" prefHeight="400" prefWidth="600" vgap="10" xmlns="http://javafx.com/javafx/11.0.1" xmlns:fx="http://javafx.com/fxml/1" fx:controller="employment.system.controllers.RegisterController">
```
Comment (Chupa-panka) [inline src/main/resources/register.fxml]:
```suggestion
<GridPane alignment="center" hgap="10" prefHeight="400" prefWidth="600" vgap="10"
xmlns="http://javafx.com/javafx/11.0.1" xmlns:fx="http://javafx.com/fxml/1"
fx:controller="employment.system.controllers.RegisterController">
```

File: src/main/resources/register.fxml [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+    <padding>
+        <Insets bottom="25.0" left="25.0" right="25.0" top="25.0" />
+    </padding>
+    <Text id="title-text" text="          REGISTRATION" textAlignment="CENTER" GridPane.columnIndex="2" GridPane.columnSpan="2" GridPane.rowIndex="1" />
```
Comment (Chupa-panka) [inline src/main/resources/register.fxml]: This one should be fixed ```text="          REGISTRATION"```. There are lots of spaces that better not to be there.

File: pom.xml [hunk 5]
```diff
@@ -7,6 +7,28 @@
...
+            <groupId>org.openjfx</groupId>
+            <artifactId>javafx-graphics</artifactId>
+            <version>11.0.1</version>
+        </dependency>
+        <dependency>
+            <groupId>org.openjfx</groupId>
+            <artifactId>javafx-fxml</artifactId>
+            <version>11.0.1</version>
+        </dependency>
+        <dependency>
```
Comment (Chupa-panka) [inline pom.xml]: I do not think that we need this dependency here.

File: pom.xml [hunk 6]
```diff
@@ -7,6 +7,28 @@
...
+            <groupId>org.slf4j</groupId>
+            <artifactId>slf4j-simple</artifactId>
+            <version>1.7.30</version>
+        </dependency>
+        <dependency>
```
Comment (Chupa-panka) [inline pom.xml]: I do not think that we need this dependency nither.

**Review by Chupa-panka — CHANGES_REQUESTED** [review]
Please review your code.

File: pom.xml (hunk 5 above)
Comment (ancae17) [reply pom.xml]: I think that we need that.

**Review by Chupa-panka — APPROVED** [review]
Great work! Thank you.

**Pull request merged**